keps/0000-kep-template.md

```diff
@@ -0,0 +1,159 @@
+---
+kep-number: 0
+title: My First KEP
```

Comment:
s/KEP/ Kubernetes Enhancement Proposal (KEP)

Reply:
KEP stands for KUDO Enhancement Proposal, not Kubernetes Enhancement Proposal. I would also keep the title short.

Reply:
Copy past error ;-). I would still explain the abbreviation somewhere

Reply:
addressed.